### PR TITLE
badge: add Mergify badge

### PR DESCRIPTION
**Pull request by sileht**
This change adds badges for Mergify

Related: #326

### CODE REVIEW

File: doc/source/badge.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.. _Badge:
+
+=========
+ Badge
+=========
+
+Mergify propose 3 badges that you can use in your project README to inform
```
Comment (jd) [inline doc/source/badge.rst]: s/propose/offers/

README? could be in any page I guess.

File: doc/source/badge.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+contributor that your repository is managed by Mergify.
```
Comment (jd) [inline doc/source/badge.rst]: your contributors.

File: doc/source/badge.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Three styles are available `cutted`, `small` and `big`:
```
Comment (jd) [inline doc/source/badge.rst]: I'd say cut rather than cutted.

File: doc/source/badge.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+|cutted| |small| |big|
+
+In a README.md:
```
Comment (jd) [inline doc/source/badge.rst]: `README.md` maybe (with backquotes)

**Comment by sileht** [comment]
Pull-request updated, HEAD is now d7bd121dbe282866f37f32f51d3c9735c97df71f

**Pull request merged**